profiles/Ramzi-Abidi.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/69228547/233711403-42cad833-4c4f-42d2-9c43-916b256a9f1d.PNG",
+    "issueId": 1811,
+    "name": "Ramzi Abidi",
+    "username": "Ramzi-Abidi"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```